style.css

```diff
@@ -110,3 +141,228 @@ li:nth-child(2) {
   height: 400px;
   background-image: url(./images/Illustration.png);
 }
+
+/* works section */
+.works {
+  width: 100%;
+  margin-top: 100px;
+  display: grid;
+  row-gap: 50px;
+}
+
+.works .title {
+  font-family: 'Crete Round', serif;
+  font-weight: 400;
+  color: #1c1a19;
+  font-size: 30px;
+  line-height: 52px;
+  margin-top: 70px;
+}
+
+.title h2 {
+  margin-left: 30px;
```

Comment:
- [ ] [OPTIONAL] This is great fixing the overflow on this title but you seem to have added the `margin-left` that tends to break your design. kindly let's fix this. 👍